sample-apps/generic_segmentation/main.py

```diff
@@ -12,7 +12,7 @@
 import logging
 import os
 
-from lib import MyInfer, MyStrategy, MyTrain
+from lib import GenericISegGraphCut, GenericISegGraphcutColdstart, GenericISegSimpleCRF, MyInfer, MyStrategy, MyTrain
```

Comment:
may be this needs to be merge to main branch first.. ?  i think these changes are part of previous PR which is already merged

Reply:
Yes, you are right. this builds on top of previous PR. Just merged this with main and resolved pending conflicts.